.github/workflows/codeql-analysis.yml

```
@@ -17,27 +14,21 @@ jobs:
+          languages: cpp, python
+      - name: Autobuild
+        uses: github/codeql-action/autobuild@v3
+        continue-on-error: true
+      - name: Perform CodeQL Analysis
+        uses: github/codeql-action/analyze@v3
+        # Security issues должны блокировать CI
+      - name: Upload SARIF file
+        uses: github/codeql-action/upload-sarif@v3
+        if: always()
+        with:
+          sarif_file: ../codeql-database/results.sarif 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)